### ISSUE TITLE
Icon request: icon-file-css / icon-file-html / icon-file-json / icon-file-js

### ISSUE THREAD

**Issue by VoidVolker** [FortAwesome/Font-Awesome#7712]
FA already have pdf/excel icons and many brand icons - it will be logical step to add few icons for css, html, json and js files. This file type is everywhere in web.

**Comment by TheChen5** [comment]
Wouldn't HTML file be ```<i class="fa fa-file-code-o"></i>```?

**Comment by VoidVolker** [comment]
Yes, it can be used. But if need to use own icon for each? Why PDF and zip have own icons and js/json/html/css don't have? It is unfair to them.

**Comment by TheChen5** [comment]
I guess #1797 could be related...
There are lots of brackets in JS, CSS, JSON.

**Comment by tagliala** [comment]
Leaving this open, I suppose the above files are strictly related

**Comment by dustinpoissant** [comment]
I need a JSON icon specifically, please add one

**Comment by tagliala** [comment]
I'm going to remove json from this list, please open a new request for fa-file-json

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
